### PR TITLE
fix!: multichain evm approach for bridge

### PR DESCRIPTION
**Pull request by bilalcorbacioglu**
## Summary

Multichain approach for bridge

It includes Li.Fi supported chains.

### CODE REVIEW

File: src/data/multiChainsHooks.ts [hunk 1]
```diff
@@ -22,10 +23,26 @@ export const usePairsHook: UsePairsHookType = {
   [ChainId.HEDERA_TESTNET]: usePairs,
   [ChainId.NEAR_MAINNET]: useNearPairs,
   [ChainId.NEAR_TESTNET]: useNearPairs,
+  //TODO: We used usePairs for now, but we need to check following chains
+  [ChainId.ETHEREUM]: usePairs,
```
Comment (SarjuHansaliya) [inline src/data/multiChainsHooks.ts:27]: do we want to use `usePairs` or we can just use `useDummyHook` ?

File: src/utils/index.ts [hunk 2]
```diff
@@ -60,6 +76,21 @@ const ETHERSCAN_PREFIXES: { [chainId in ChainId]: string } = {
   296: CHAINS[ChainId.HEDERA_TESTNET].blockExplorerUrls?.[0] || '',
   329847900: CHAINS[ChainId.NEAR_MAINNET].blockExplorerUrls?.[0] || '',
   329847901: CHAINS[ChainId.NEAR_TESTNET].blockExplorerUrls?.[0] || '',
+  1: '',
+  137: '',
+  66: '',
+  106: '',
+  56: '',
+  42220: '',
```
Comment (SarjuHansaliya) [inline src/utils/index.ts:84]: in this mapping, can we use [ChainId.ETHEREUM] instead of direct number mapping .... like we are doing for every mapping

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]
can we use `useDummyHook` instead of actual hook at below places ?

- `usePairs`
- `useWrapCallback`
- `useApproveCallbackFromTrade`
- `useSwapCallback`
- `useApproveCallback`
- `useTokenBalances`
- `useEVMPairBalance`
- `useAddLiquidity`
- `useRemoveLiquidity`
- `useGetUserLP`

File: src/utils/index.ts (hunk 2 above)
Comment (bilalcorbacioglu) [reply src/utils/index.ts]: fixed

**Pull request merged**